pydicom/cli/show.py

```diff
@@ -0,0 +1,176 @@
+# Copyright 2019 pydicom authors. See LICENSE file for details.
+"""Pydicom command line interface program for `pydicom show`"""
+
+from pydicom import dcmread
+from pydicom.data.data_manager import get_testdata_file
+from pydicom.dataset import Dataset
+import sys
+
+from pydicom.cli.main import filespec_help, filespec_parser
+
+
+def add_subparser(subparsers):
+    subparser = subparsers.add_parser(
+        "show", description="Display all or part of a DICOM file"
+    )
+    subparser.add_argument(
+        "filespec", help=filespec_help, type=filespec_parser
+    )
+    subparser.add_argument(
+        "-x",
+        "--exclude-private",
+        help="Don't show private data elements",
+        action="store_true",
+    )
+    subparser.add_argument(
+        "-t", "--top", help="Only show top level", action="store_true"
+    )
+    subparser.add_argument(
+        "-q",
+        "--quiet",
+        help="Only show basic information",
+        action="store_true",
+    )
+
+    subparser.set_defaults(func=do_command)
+
+
+def do_command(args):
+    if len(args.filespec) != 1:
```

Comment:
For future, it would be nice to support multiple files or wildcards, forexample to compare some specific tag values in many files (though maybe you don't want to duplicate `dcmdump` functionality...)

Reply:
Yeah, the intent is (for some commands at least) to allow wildcards.  At this point I just wanted to return the list and modify the command code as little as possible since I've let this slide a couple of weeks already.